### PR TITLE
[ENG-617] Contributor management cohesive design

### PR DESCRIPTION
**Pull request by adlius**
- Ticket: [ENG-617]
- Feature flag: n/a

## Purpose

Various CSS and template fixes for better UI on mobile view.

## Summary of Changes

Added `bibliographicContributors` relationship to the `draft-registration` model. Note: the API already serialize this relationship.
Change `review-metadata-renderer` so that it passes `draftRegistration` instead of `node` to the `contributors-list` component.

Added the dropdown for institution and education info to the editable card.

Screenshot from 2021-03-01 16-20-09

## QA Notes

[ENG-617]: https://openscience.atlassian.net/browse/ENG-617

### CODE REVIEW

**Comment by fabmiz** [comment]
@adlius merge conflicts 👀

File: lib/osf-components/addon/components/contributors/card/editable/template.hbs [hunk 1]
```diff
@@ -51,7 +51,22 @@
                 {{on 'click' (fn @manager.toggleContributorIsBibliographic @contributor)}}
             />
         </span>
-        <span local-class='CardSectionSmall'>
+        <span
```
Comment (brianjgeiger) [inline lib/osf-components/addon/components/contributors/card/editable/template.hbs:54]: Was this only on the read-only card before, not the editable card? If so, this seems like a good thing to add to the "changes" section of the PR description. If not, I'm very confused why it's here.

**Review by brianjgeiger — CHANGES_REQUESTED** [review]
Overall, looks good. As Fabrice said, there's a merge conflict, and I wouldn't be surprised if there weren't more with the draft-node stuff coming along. I was a little confused about one of the changes, but I think I figured it out (see below).

File: lib/osf-components/addon/components/contributors/card/editable/component.ts [hunk 2]
```diff
@@ -15,4 +17,10 @@ export default class ContributorsCardEditable extends Component {
 
     // private properties
     permissionOptions = Object.values(Permission);
+    @tracked showDropdown = false;
+
+    @action
+    toggleDropdown() {
+        this.toggleProperty('showDropdown');
```
Comment (fabmiz) [inline lib/osf-components/addon/components/contributors/card/editable/component.ts:24]: with `showDropdown` being `tracked`, you might want to reassign as such
```suggestion
        this.showDropdown = !this.showDropdown;
```
`toggleProperty` uses  legacy `@ember/component` `set()` underneath

File: lib/osf-components/addon/components/contributors/card/editable/template.hbs (hunk 1 above)
Comment (adlius) [reply lib/osf-components/addon/components/contributors/card/editable/template.hbs:54]: Correct. This was not in the editable cards. I've added to the changes in the PR description.

**Comment by brianjgeiger** [comment]
There are still some failing tests that look relevant. Probably just test setup with the new "contributors on draft instead of node" workflow.

**Review by brianjgeiger — APPROVED** [review]

**Review by futa-ikeda — APPROVED** [review]
🚀

**Pull request merged**